static/app/views/organizationCreate/index.tsx

```diff
@@ -1,75 +1,60 @@
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t, tct} from 'sentry/locale';
 import ConfigStore from 'sentry/stores/configStore';
-import {OrganizationSummary} from 'sentry/types';
+import {OrganizationSummary, Region} from 'sentry/types';
+import {
+  getRegionBaseUrl,
+  getRegionByName,
+  getRegionChoices,
+  shouldDisplayRegions,
+} from 'sentry/utils/regions';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
-enum RegionDisplayName {
-  US = '🇺🇸 United States of America (US)',
-  DE = '🇪🇺 European Union (EU)',
-}
-
-function getRegionChoices(): [string, string][] {
-  const regions = ConfigStore.get('regions') ?? [];
-
-  return regions.map(({name, url}) => {
-    const regionName = name.toUpperCase();
-    if (RegionDisplayName[regionName]) {
-      return [url, RegionDisplayName[regionName]];
-    }
-
-    return [url, name];
-  });
-}
-
-function getDefaultRegionChoice(
-  regionChoices: [string, string][]
-): [string, string] | undefined {
-  if (!shouldDisplayRegions()) {
-    return undefined;
-  }
-
-  const usRegion = regionChoices.find(
-    ([_, regionName]) => regionName === RegionDisplayName.US
-  );
-
-  if (usRegion) {
-    return usRegion;
-  }
-
-  return regionChoices[0];
-}
-
-function shouldDisplayRegions(): boolean {
-  const regionCount = (ConfigStore.get('regions') ?? []).length;
-  return (
-    ConfigStore.get('features').has('organizations:multi-region-selector') &&
-    regionCount > 1
-  );
-}
-
-function removeRegionFromRequestForm(formData: Record<string, any>) {
-  const shallowFormDataCopy = {...formData};
-
-  delete shallowFormDataCopy.region;
-  return shallowFormDataCopy;
+function removeDataStorageLocationFromFormData(
+  formData: Record<string, any>
+): Record<string, any> {
+  const shallowFormDataClone = {...formData};
+  delete shallowFormDataClone.dataStorageLocation;
+  return shallowFormDataClone;
 }
 
 function OrganizationCreate() {
   const termsUrl = ConfigStore.get('termsUrl');
   const privacyUrl = ConfigStore.get('privacyUrl');
   const regionChoices = getRegionChoices();
-  const [regionUrl, setRegion] = useState<string | undefined>(
-    getDefaultRegionChoice(regionChoices)?.[0]
-  );
+  const [region, setRegion] = useState<Region | undefined>(undefined);
+
+  const formModel: FormModel = new FormModel({
+    apiOptions: {
+      baseUrl: getRegionBaseUrl(region),
+    },
+    transformData: removeDataStorageLocationFromFormData,
+  });
+
+  let dataStorageSelect: ReactElement | null = null;
+  if (shouldDisplayRegions()) {
+    dataStorageSelect = (
+      <SelectField
+        name="dataStorageLocation"
+        label="Data Storage Location"
+        help="Where will this organization reside?"
+        choices={regionChoices}
+        onChange={regionName => setRegion(getRegionByName(regionName))}
+        inline={false}
+        stacked
+        required
+      />
+    );
+  }
```

Comment:
Any reason you moved this up out of the jsx below?

not really a problem, I just dislike `let`'s haha

Reply:
Nope, it was leftover from a previous refactor attempt. Reverted it back to being inlined.